packages/styles/scss/patterns/blocks/leadspace-block/_leadspace-block.scss

```diff
@@ -47,10 +39,10 @@
       }
 
       @include carbon--breakpoint('sm') {
-        padding-bottom: $layout-01;
+        padding-bottom: $layout-06;
       }
       @include carbon--breakpoint('md') {
-        padding-bottom: $layout-06;
+        padding-bottom: $layout-07;
       }
```

Comment:
These breakpoints should be `md` and `lg`, respectively. There actually is no `sm`.

Reply:
Thank you @kennylam! I did not realize that they needed to be in order, but that makes a lot of sense now.